generator/integration_tests/golden/unit_tests/.clang-format

```diff
@@ -0,0 +1,36 @@
+# Use the Google style in this project.
+BasedOnStyle: Google
```

Comment:
I assume the intent is to have this file match our top-level `.clang-format` file. If that's correct, it would be nice to have some way to know if they diverge. How about adding a simple comparison check in `ci/check-style.sh` that breaks our `clang-tidy` build if this file differs from the top-level `.clang-format` file?